android/src/main/kotlin/com/hypertrack/sdk/flutter/HyperTrackPlugin.kt

```diff
@@ -82,25 +88,34 @@ public class HyperTrackPlugin(): FlutterPlugin, MethodCallHandler, StreamHandler
       result.error("NOT_INITIALIZED", "Internal Error: onMethodCall(${call.method}) - sdkInstance is null", null)
       return
     }
-
+    
     when (call.method) {
       "getDeviceId" -> result.success(sdk.deviceID)
       "isRunning" -> result.success(sdk.isRunning)
+      "isTracking" -> result.success(sdk.isTracking)
+      "getAvailability" -> result.success(sdk.availability.toString())
+      "setAvailability" -> setAvailability(result,sdk,call.arguments<Boolean>()?:false)
```

Comment:
should this default to `false`?

Reply:
@AshishSynoriq  There should be no defaults. `setAvailablility` on Dart should accept a non-null value and pass it here.